shell/browser/api/electron_api_session.cc

```diff
@@ -138,8 +138,6 @@ uint32_t GetQuotaMask(const std::vector<std::string>& quota_types) {
     auto type = base::ToLowerASCII(it);
     if (type == "temporary")
       quota_mask |= StoragePartition::QUOTA_MANAGED_STORAGE_MASK_TEMPORARY;
-    else if (type == "persistent")
-      quota_mask |= StoragePartition::QUOTA_MANAGED_STORAGE_MASK_PERSISTENT;
```

Comment:
need to update docs/api/session.md

Reply:
@nornagon do you know where in that doc we need to update?

Reply:
ah nvm... I found it.